[ao] Adding FAQ to docs

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #87322

Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

HDCharles committed:
[ao] Adding FAQ to docs

Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] Adding FAQ to docs"


Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

z-a-f commented on docs/source/quantization.rst:
```diff
@@ -998,6 +998,30 @@ if ``dtype`` is ``torch.qint8``, make sure to set a custom ``quant_min`` to be `
+   Supporting export to onnx model is not a priority for PyTorch quantization, please open an issue in `GitHub - onnx/onnx <https://github.com/onnx/onnx>`_  when you encounter problems with ONNX,
+   or reach out to people in this list: `PyTorch Governance | Maintainers | ONNX exporter <https://pytorch.org/docs/stable/community/persons_of_interest.html#onnx-exporter>`_
+
+3. How can use quantization with LSTM's?:
```
nit: Is a word missing? Seems awkward

HDCharles replied:
good catch

z-a-f commented on docs/source/quantization.rst:
```diff
@@ -998,6 +998,30 @@ if ``dtype`` is ``torch.qint8``, make sure to set a custom ``quant_min`` to be `
+
+2. Where can I get ONNX support for my quantized model?:
+
+   Supporting export to onnx model is not a priority for PyTorch quantization, please open an issue in `GitHub - onnx/onnx <https://github.com/onnx/onnx>`_  when you encounter problems with ONNX,
```
nit: IMHO, saying that it's not a priority sounds a little dismissive. Could we reword it?

HDCharles replied:
yeah, i've changed the language to omit that part

z-a-f reviewed (approved):
I think we can always add more items here, if some questions keep on popping up

HDCharles committed:
Update on "[ao] Adding FAQ to docs"


Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] Adding FAQ to docs"


Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] Adding FAQ to docs"


Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] Adding FAQ to docs"


Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] Adding FAQ to docs"


Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles commented:
@pytorchmergebot merge

HDCharles committed:
Update on "[ao] Adding FAQ to docs"


Summary: migrated from: https://discuss.pytorch.org/t/quantization-frequently-asked-questions/161251

Test Plan: circle CI tests

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles commented:
@pytorchmergebot merge
